software/stormm.yaml

```
@@ -0,0 +1,13 @@
+name: STORMM
+description: "A codebase that creates a new family of molecular dynamics programs that use shared C++ classes and GPU kernels to provide accessible, high-performance simulations across both CPU and GPU platforms with minimal performance trade-offs."
+docs: https://psivant.github.io/stormm
```

```suggestion
docs: https://stormm.psivant.com/doxygen/index.html
```

previous link is 404